packages/api-client/src/index.ts

```diff
@@ -13,3 +13,5 @@ export { AuthHandler } from './types';
 export * from './auth';
 
 export * from './constants';
+
+export * from './__tests__/mockConnections';
```

Comment:
Tried a little bit to see if it's possible to [just export these when in `development` mode](https://webpack.js.org/guides/package-exports/#optimizations). But it was proving a bit tricky with how we resolve local packages in yarn. If someone knows how to do this I think it'd be nice, but seems not a big deal to me?

Reply:
We shouldn't ever be putting __tests__ into the dist folder 😄 

MockApiClient isn't actually a test, it's a utility that can be used for testing, like a dry run flag in a lib to use an analogy, it's a real part of the lib.

Reply:
Oh so you think I should move it into somewhere else in the `src` directory? I guess I had felt since they're just for testing they should be under the `__tests__` directory.

But I could see them fitting under `src/connections/mocks`, seems appropriate 👍